frontend/yarn.lock

```diff
@@ -8021,6 +8085,21 @@ kind-of@^6.0.0, kind-of@^6.0.2:
   version "6.0.2"
   resolved "https://registry.yarnpkg.com/kind-of/-/kind-of-6.0.2.tgz#01146b36a6218e64e58f3a8d66de5d7fc6f6d051"
 
+kubevirt-web-ui-components@^0.1.34-2:
```

Comment:
I'm concerned with all the additional dependencies this is bringing in. It looks like it includes some alternative libraries to things we've already included in console. Is this only being used for the `getName` / `getNamespace` / `getUid` helpers?

I realize you use it in the kubevirt fork for other things.

Reply:
As an alternative, we could move the code we need from `kubevirt-web-ui-components` to `console-shared` (or create another, more focused shared package) and align it with current Console dependencies, code (like `Firehose` vs. `WithResources`) etc.

Reply:
> Is this only being used for the `getName` / `getNamespace` / `getUid` helpers?

+ a couple of things in machine-cell.tsx

I can quite easily remove this dependency for metal3 plugin in general as it does not have large amount of parts implemented in web-ui-components. For kubevirt plugin It will be more difficult.

Reply:
With recent change I've removed the kubevirt/web-ui-components dependency and added required selectors to @console/shared package.